test/e2e/util.go

```diff
@@ -482,35 +539,49 @@ func RunRC(c *client.Client, name string, ns, image string, replicas int) error
 		return fmt.Errorf("Error creating replication controller: %v", err)
 	}
 
+	// Create a routine to query for the list of pods
+	stop := make(chan struct{})
+	go func(stop <-chan struct{}, n string, ns string, l labels.Selector) {
+		for {
+			select {
+			case <-stop:
+				return
+			default:
+				p, err := c.Pods(ns).List(l, fields.Everything())
+				if err != nil {
+					Logf("Warning: Failed to get pod list: %v", err)
+				} else {
+					podLists.Push(p.Items)
+				}
+				time.Sleep(1 * time.Second)
```

Comment:
@fgrzadkowski It looks like the watch tools are more for real time monitoring.  Am I missing something?  If that's the case then I think we could use it if we can record timestamps for events but we'll have to do some processing in the test to get figure out a 1 second granularity.  It would probably also change the way we process the pod states.  I would rather we keep this as is and look at using a watch as a separate pr or if there is a performance issue.